ocs_ci/ocs/utils.py

```diff
@@ -1003,6 +1048,46 @@ def collect_prometheus_metrics(
             json.dump(datapoints.json(), outfile)
 
 
+def save_live_logs(request, pods_containers_dict, patterns_to_log=None):
+    """
+    Save pods logs to files
+
+    """
+    tc_name = request.node.name
+
+    def finalizer():
+        """
+        Stop the thread that executed save_logs()
+
+        """
+        ocsci_config.RUN["save_live_pod_logs"] = False
+        if thread:
+            thread.join()
+
+    request.addfinalizer(finalizer)
+
+    ocsci_config.RUN["save_live_pod_logs"] = True
+
+    def save_logs():
+        while ocsci_config.RUN.get("save_live_pod_logs"):
+            time.sleep(10)
```

Comment:
Maybe we can add here a var with a name like `seconds_to_wait_between_save_logs = 10`. Currently, it's not so clear why we need to wait 10 seconds.

Reply:
sure, I will add this var and also a comment explaining why we wait

Reply:
In second thought, maybe it is not necessary to wait at all between iterations.
I will try without any sleep